cumulusci/tasks/datadictionary.py

```diff
@@ -122,6 +156,22 @@ def _init_options(self, kwargs):
                     "Only GitHub dependencies are currently supported."
                 )
 
+        self.options["include_prerelease"] = process_bool_arg(
+            self.options.get("include_prerelease") or False
```

Comment:
Slightly more concise 🤷‍♂️ 
```suggestion
            self.options.get("include_prerelease", False)
```

Reply:
`process_bool_arg()` documents that `None` is accepted as falsy but that behavior is deprecated, so this construct ensures we don't pass an explicit or implicit None.